docs/docusaurus/docs/core/introduction/about_gx.md

```diff
@@ -62,27 +66,10 @@ GX is NOT a data versioning tool.
 
 GX does not store data itself. Instead, it deals in metadata about data: Expectations, Validation Results, etc. If you want to bring your data itself under version control, check out tools like: [DVC](https://dvc.org/), [Quilt](https://github.com/quiltdata/quilt), and [lakeFS](https://github.com/treeverse/lakeFS/).
 
-GX currently works best in a Python environment.
+GX is NOT an independent executable.
 
-GX is Python-based. You can invoke it from the command line without using a Python programming environment, but if you’re working in another ecosystem, other tools might be a better choice. If you’re running in a pure R environment, you might consider [assertR](https://github.com/ropensci/assertr)  as an alternative. Within the TensorFlow ecosystem, [TFDV](https://www.tensorflow.org/tfx/guide/tfdv) fulfills a similar function as GX.
+{GxData.product_name} is a Python library.  To use GX, you will need an installation of <PythonVersion/>.  Ideally, you will also configure a Python virtual environment to use GX from.  Guidance on setting up your Python environment and installing the GX Python library is provided under [Set up a GX environment](/core/installation_and_setup/installation_and_setup.md) in the GX docs.
```

Comment:
> _Ideally, you will also configure a Python virtual environment to use GX from._

Just a nitpicky language comment here (feel free to tweak/incorporate or leave the text as-is) - I'd suggest:

_Ideally, you will also configure a Python virtual environment in which you can install and run GX._